pytest/lib/cluster.py

```diff
@@ -780,10 +780,15 @@ def spin_up_node_and_push(i, boot_node: BootNode):
 ROOT_DIR = pathlib.Path(__file__).resolve().parents[2]
 
 DEFAULT_CONFIG = {
-    'local': True,
-    'near_root': os.environ.get("NEAR_ROOT", str(os.path.join(ROOT_DIR, 'target/debug'))),
-    'binary_name': 'neard',
-    'release': False,
+    'local':
+        True,
+    'near_root':
+        os.environ.get("NEAR_ROOT", str(os.path.join(ROOT_DIR,
+                                                     'target/debug'))),
```

Comment:
I mean, just add `NEAR_ROOT` global variable at the top so this doesn’t have to wrap. ;)

Reply:
Thanks, done.